buddybuild_prebuild.sh

```diff
@@ -2,4 +2,9 @@
 echo "Setting SentryDSN to $SENTRY_DSN"
 /usr/libexec/PlistBuddy -c "Set SentryDSN $SENTRY_DSN" "lockbox-ios/Common/Resources/Info.plist"
 
+if ! [ -x "$(command -v swiftlint)" ] ; then
```

Comment:
@sashei last question before landing...I was having [BB error ]([url](https://dashboard.buddybuild.com/apps/5a0ddb736e19370001034f85/build/5cc6c664fde6cb00016f9b75#logs))with swift already installed, 
```
    Error: swiftlint 0.27.0 is already installed
    To upgrade to 0.32.0, run `brew upgrade swiftlint`.
```
it may be only an issue until may 6th when they stop installing it...but at least as a workaround until then I changed this script...is that fine or do you prefer a better solution? Thanks!